src/Features/VisualBasic/Portable/CodeLens/DisplayInfoVisualBasicServices.vb

```diff
@@ -0,0 +1,275 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+Imports System.Composition
+Imports System.Globalization
+Imports Microsoft.CodeAnalysis.CodeLens
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeLens
+    <ExportLanguageService(GetType(IDisplayInfoLanguageServices), LanguageNames.VisualBasic), [Shared]>
+    Friend NotInheritable Class DisplayInfoVisualBasicServices
+        Implements IDisplayInfoLanguageServices
+
+        Private Shared ReadOnly DefaultDisplayFormatVB As SymbolDisplayFormat = New SymbolDisplayFormat(
```

Comment:
I'm afraid this code predates the time i started on it.. perhaps srivatsn or jasonmal might know.